libs/librepcb/common/fileio/filepath.cpp

```diff
@@ -260,16 +260,16 @@ QString FilePath::cleanFileName(const QString& userInput, CleanFileNameOptions o
 {
     // perform compatibility decomposition (NFKD)
     QString ret = userInput.normalized(QString::NormalizationForm_KD);
+    // remove all invalid characters (note: backslashes are required to escape minus sign)
+    ret.remove(QRegularExpression("[^ _\\-.0-9A-Za-z]"));
```

Comment:
Don't you need to escape the '.' as well? '.' means any character, doesn't it? Or is this different inside a `[]` group?